mitiq/interface/mitiq_qiskit/conversions.py

```diff
@@ -249,14 +249,19 @@ def from_qiskit(circuit: qiskit.QuantumCircuit) -> cirq.Circuit:
     Returns:
         Mitiq circuit representation equivalent to the input Qiskit circuit.
     """
+
     try:
         mitiq_circuit = from_qasm(qasm2.dumps(circuit))
     except QasmException:
         # Try to decompose circuit before running
         # This is necessary for converting qiskit circuits with
         # custom packaged gates, e.g., QFT gates
-        circuit = circuit.decompose()
+        gates_to_decompose = ["rxx", "rzz", "rzx", "ryy", "QFT"]
```

Comment:
nit: this should be capitalized as we want to treat it as a constant

Reply:
I have updated the code. Thank you!